### ISSUE TITLE
05-encode.t fails with Encode 3.09

### ISSUE THREAD

**Issue by jplesnik** [creaktive/Test-Mojibake#15]
```
#   Failed test 'Mojibake test for /usr/lib64/perl5/vendor_perl/Encode.pm'
#   at /builddir/build/BUILD/Test-Mojibake-1.3/blib/lib/Test/Mojibake.pm line 176.
# UTF-8 unexpected in /usr/lib64/perl5/vendor_perl/Encode.pm, line 492 (POD)
# Looks like you failed 1 test of 3.
t/05-encode.t ................ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```
The tests starts failing after updating the module Encode to 3.09

**Comment by creaktive** [comment]
Thanks for reporting!
Looking into the issue, I found that UTF-8 apostrophes were introduced in the POD section of [Encode.pm in version 3.09](https://metacpan.org/diff/file?target=DANKOGAI%2FEncode-3.09%2F&source=DANKOGAI%2FEncode-3.08). [perlpod states](https://perldoc.perl.org/perlpod#=encoding-encodingname) that _"It's best to be explicit if you use anything besides strict ASCII"_; so technically, Encode.pm itself suffers from a mild case of mojibake... I'll report this upstream for the time being.

**Comment by creaktive** [comment]
https://rt.cpan.org/Ticket/Display.html?id=136534

**Comment by dankogai** [comment]
Thank you.  Fixed in https://github.com/dankogai/p5-encode/commit/5cedcc336fdc92fbd962d828b63cafe42a5c5a32 .  `Encode.pm` itself should contain only ASCII.

**Comment by jplesnik** [comment]
It was solved by Encode 3.10.
